css/_toc.scss

```diff
@@ -3,9 +3,16 @@
 // Common ToC styles shared by pre-2.3 static ToCs & dynamic ToCs
 // Select <nav> element based on new (Bootstrap 4) column classes
 
-nav.col-lg-3 {
```

Comment:
As the comment above explains, the docs for earlier (pre-2.3) versions rely on the presence of this class.

Removing it breaks the ToC on older docs versions, which we don't want to regenerate.
See https://deploy-preview-411--dita-ot.netlify.app/2.2/release-notes/ for example.

Reply:
Yeah, forgot about that. I really dislike that we used `col-lg-3` directly back when we added that, instead of a custom class that extends it. Oh well, have to revert that for backwards compatibility.

Reply:
Moving these changes to the block that targets only the newer [dynamic ToC](https://github.com/dita-ot/website/blob/feature/scroll-to-start/css/_toc.scss#L45-L100) should be safer.